Xamarin.Forms.Controls/CoreGalleryPages/LabelCoreGalleryPage.cs

```diff
@@ -43,9 +43,9 @@ protected override void Build (StackLayout stackLayout)
 #pragma warning restore 618
 
 			var formattedString = new FormattedString ();
-			formattedString.Spans.Add (new Span { BackgroundColor = Color.Red, ForegroundColor = Color.Olive, Text = "Span 1 " });
-			formattedString.Spans.Add (new Span { BackgroundColor = Color.Black, ForegroundColor = Color.White, Text = "Span 2 " });
-			formattedString.Spans.Add (new Span { BackgroundColor = Color.Pink, ForegroundColor = Color.Purple, Text = "Span 3" });
+			formattedString.Spans.Add (new Span { BackgroundColor = Color.Red, TextColor = Color.Olive, Text = "Span 1 " });
```

Comment:
These are changed because the ForegroundColor is now obsolete, as per Stephane's request.